frontend/src/common/styles.js

```diff
@@ -4,35 +4,40 @@ import Colors from './colors';
 
 export default StyleSheet.create({
   container: {
-    padding: 40,
     flex: 1,
+    padding: 10,
+    paddingTop: Constants.statusBarHeight,
     backgroundColor: Colors.lightenGrey,
-    alignItems: 'center',
   },
   textStyle: {
-    color: '#55565a',
+    color: Colors.textColor,
     fontSize: 15,
     lineHeight: 24,
     letterSpacing: 0.15,
   },
   cardStyle: {
-    marginBottom: 20,
-    paddingTop: 12,
-    paddingHorizontal: 12,
-    paddingBottom: 13,
-    display: 'flex',
-    flexDirection: 'row',
-    alignItems: 'center',
-    borderRadius: 10,
-    elevation: 2,
-    backgroundColor: '#10978b',
+    shadowColor: '#000',
```

Comment:
if we store the white color in the common colors, I think we can store the black too, opinion?

Reply:
Yes, of sure